chia/full_node/coin_store.py

```diff
@@ -364,8 +364,7 @@ async def get_coin_state_by_ids(
         self,
         include_spent_coins: bool,
         coin_ids: List[bytes32],
-        start_height: uint32 = uint32(0),
-        end_height: uint32 = uint32((2 ** 32) - 1),
+        min_height: uint32 = uint32(0),
```

Comment:
I notice there's no unit test for this function, in `tests/core/full_node/test_coin_store.py`. It would be good to have one

Reply:
Thank you, yes that would be good considering it's in the coin store.